src/runtime_src/core/edge/common_em/rpc_messages.proto

```diff
@@ -498,7 +498,7 @@ message xclGraphEnd_call {
   required uint32 gh = 2;  
 }
 message xclGraphEnd_response {
-     required bool ack = 1;
+  required uint32 ack = 1;
```

Comment:
Add a comment here about what is the semantics of this value, i.e. describes your protocol : 1 is for..., 0 is for..., -1 is for...

Reply:
Added the comments related to ack in the RPC call.